streamz/tests/test_core.py

```diff
@@ -787,6 +787,25 @@ def test_from_file():
                 assert time() < start + 2  # reads within 2s
 
 
+@gen_test()
+def test_from_file_end():
+    with tmpfile() as fn:
+        with open(fn, 'wt') as f:
+            f.write('data1\n')
+            f.flush()
+
+            source = Stream.from_textfile(fn, poll_interval=0.010,
+                                          start=False, from_end=True)
+            out = source.sink_to_list()
+            source.start()
+            assert out == []
+            yield gen.sleep(0.01)
```

Comment:
A little nit-picky, but why this sleep?

Reply:
The `seek()` to end doesn't happen until the coroutine runs for the first time, so if you were to write immediately here, you would always seek past the new data. This yield makes sure the coroutine runs once. Would a yield without the sleep have the same effect, or is there another way to say "run coroutines now"?

Reply:
If you just want one cycle to go through, then try `yield gen.moment`.

If you might need a few cycles, then you should probably have an `await_for`

Reply:
I'll try `moment` tomorrow; `await_for` is tricky when you *don't* expect the output to change. Perhaps it would be reasonable to await on at attribute of the source.